src/utils/graphql-client/graphql-client.ts

```diff
@@ -66,19 +67,30 @@ export const createResponseLink = (config: ResponseLinkConfig): ApolloLink =>
     });
   });
 
-export const createHttpLink = (uri: string): ApolloLink =>
-  createApolloHttpLink({
-    uri,
+export const createHttpLink = (options?: HttpOptions): ApolloLink => {
+  return createApolloHttpLink(options);
+};
+
+export const switchUriLink = (uri: string): ApolloLink =>
+  new ApolloLink((operation, forward) => {
+    const { projectVid } = operation.getContext();
+    operation.setContext({
+      uri: `${uri}${projectVid || ''}`,
+    });
+
+    return forward(operation);
   });
 
 export function createGraphqlClient(config: Config): GraphQLClient {
   const { uri, identity, onError: handleError } = config;
+  const uriLink = switchUriLink(uri);
   return new ApolloClient({
     cache: new InMemoryCache(),
     link: from([
       createResponseLink({ handleError }),
       createErrorLink({ handleError }),
-      createAuthLink(identity).concat(createHttpLink(uri)),
+      uriLink,
```

Comment:
```suggestion
      createSwitchUriLink(uri),
```
давай тут тоже к одному виду приведем